lambda/main.go

```diff
@@ -110,6 +120,24 @@ func LambdaHandler(event NewRelease) error {
 	return nil
 }
 
+func isPreRelease(version string) (bool, error) {
```

Comment:
can we not use https://pkg.go.dev/golang.org/x/mod/semver#Prerelease ?

Reply:
Supposedly the package I referenced supports the entire spec. I've never used either before.